pages/internorientation.md

```diff
@@ -24,6 +24,7 @@ Note that this will give you the ability to make bigger mistakes - so make sure
 You will now have the ability to review, close, and merge pull requests and issues.
 ### Reviewing pull requests and issues
 One of your responsibilities as an intern is to review and help prospective interns with their issues / pull request on open learning exchange. You can start a review by going to the files changed tab on a pull request. You can read more about [reviewing on Github here](https://help.github.com/articles/about-pull-request-reviews/). You should check for things such as:
+* [x] Check for issue number in pull request title
 * [x] Is there any unneeded files in the pull request?
```

Comment:
@Loshma93 Minor issue (Not related to your changes): I have noticed that the second check-list should be "Are there any unnecessary files in the pull request". Could we use this PR to fix that as well ?